### ISSUE TITLE
`close_all` is not correctly intercepted

### ISSUE THREAD

**Issue by SCdF** [facelessuser/TabsExtra#56]
To reproduce:
 - Have a sticky tab and a bunch of other tabs
 - Close all tabs using a keyboard shortcut:
```
    { "keys": ["super+ctrl+shift+w"], "command": "close_all"},
```
 - Or using File -> Close All Files

And note that the stickied tab gets closed as well. It still works if you right click a tab and select "Close all tabs".

MacOSX. ST3 dev version, though I've noticed it in many versions prior to this as well.

**Comment by Krato** [comment]
+1

**Comment by facelessuser** [comment]
I totally missed this issue. Haven't had a chance to touch this is a bit tough. I'm surprised that somehow this command is getting missed. I'll have to confirm this and look into fixing if I can reproduce.

**Comment by facelessuser** [comment]
This will be fixed in the next release. Somehow, a line of code made it in which broke our overrides. It may be that it was trying to solve some other corner case, but unfortunately, I cannot recall what corner case that may be, and frankly, it was solving something, it was implemented in a bad way. For now I've removed it and close_all overrides are working in all situation again. I'll keep my eye out and see if any other issues crop up related to this fix, and if so, will implement a fix in a more appropriate way.
